app/decorators/article_decorator.rb

```diff
@@ -1,8 +1,6 @@
-class ArticleDecorator < Draper::Decorator
+class ArticleDecorator < BaseDecorator
   decorates_association :author
-  delegate :title, :content, :created_at, :updated_at, :author_id
-  # How to clearly use Decorated objects with forms?
-  delegate :to_key, :persisted?, :errors
+  delegate :id, :title, :content, :author_id
```

Comment:
there should be "to: " param?

and I think it's not good idea to delegate ID. I understand that you want to build forms using decorated object, but there is another way to do it. We can discuss it later44